packs/spell-effects/spell-effect-fortissimo-composition.json

```
@@ -0,0 +1,96 @@
+{
+    "_id": "rAW5o8MjjuIhYhLg",
+    "img": "systems/pf2e/icons/spells/inspire-heroics.webp",
+    "name": "Spell Effect: Fortissimo Composition",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Fortissimo Composition]</p>\n<p>Increase the status bonus from the composition to +2 (+3 on a critical success).</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
+                        "value": "3"
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.success",
+                        "value": "2"
+                    }
```

```suggestion
                    {
                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
                        "value": 3
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.success",
                        "value": 2
                    }
```
Making these explicitly numbers, and not strings that look like numbers.